### DISCUSSION TITLE
Unable to change WebRadio cover art after 20210528 update

### DISCUSSION THREAD

**Discussion by ScottVinzant** [rern/rAudio#347] [Fixed Issues]
After the 20210528 update, I am no longer able to change or update the cover art for web radio stations.
See attached screenshot.
![Screenshot from 2021-05-29 11-18-35](https://user-images.githubusercontent.com/84992585/120077397-e45af080-c06f-11eb-91d6-e0d4e27414ae.png)

**Comment by rern** [comment]
Fixed with today's update.

**Reply by ScottVinzant** [reply]
Verified.
